iaslic-bulletin.csl

```diff
@@ -0,0 +1,397 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB">
+  <info>
+    <title>IASLIC Bulletin</title>
+    <id>http://www.zotero.org/styles/iaslic-bulletin</id>
+    <link href="http://www.zotero.org/styles/iaslic-bulletin" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-with-titles" rel="template"/>
+    <link href="https://www.iaslic1955.org.in/fckeditor/userfiles/file/Publication_policy_Authorship_Manual_final.pdf" rel="documentation"/>
+    <author>
+      <name>Dr. Aditya Tripathi</name>
+      <email>aditya@bhu.ac.in</email>
+    </author>
+    <author>
+      <name>Nikhil Anand Srivastava</name>
+      <email>nikhil@bhu.ac.in</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <category field="humanities"/>
+    <issn>0018-8441</issn>
+    <summary>IASLIC citation style for Library and Information Science research articles, version 1.</summary>
+    <updated>2025-07-31T07:47:03+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="retrieved">Retrieved on</term>
+      <term name="volume" form="short">V</term>
+      <term name="editor" form="short">
+        <single>Ed.</single>
+        <multiple>Ed.</multiple>
+      </term>
+      <term name="page" form="short">p</term>
```

Comment:
You can achieve this output where you call the page label by supressing the punctuation and setting plural to never. Please remove these from here.